mobile/src/src/components/Housing/Listings/ListingView.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { StyleSheet, Dimensions, Image, View, Text } from "react-native";
+import { Card, Title, Paragraph, Avatar } from 'react-native-paper';
+import ProfilePhoto from "../../common/ProfilePhoto";
+import { Choices } from "../../../utils/endpoints";
+
+const ListingView = (props) => {
+
+    const {width, height} = Dimensions.get('window');
+
+    return (
+        <Card style={styles.container}>
+            <Card.Content>
+                <View style={{ flexDirection: "row" }}>
```

Comment:
We can probably add this style to styles variable. Then we can reuse it for the ones below.